tests/meta/orders/test_meta_orders_api.py

```diff
@@ -85,33 +114,25 @@ def test_submit(
                 assert link_obj.sample.original_ticket == ticket_id
 
 
-def monkeypatch_process_lims(monkeypatch, order_data):
-    lims_project_data = {"id": "ADM1234", "date": dt.datetime.now()}
-    lims_map = {sample.name: f"ELH123A{index}" for index, sample in enumerate(order_data.samples)}
-    for submitter in SUBMITTERS:
-        monkeypatch.setattr(
-            f"cg.meta.orders.{submitter}.process_lims",
-            lambda **kwargs: (lims_project_data, lims_map),
-        )
-
-
 @pytest.mark.parametrize(
     "order_type",
     [OrderType.MIP_DNA, OrderType.MIP_RNA, OrderType.BALSAMIC],
 )
+@patch("cg.clients.freshdesk.freshdesk_client.FreshdeskClient.create_ticket")
 def test_submit_ticketexception(
+    mock_create_ticket,
     all_orders_to_submit,
     orders_api: OrdersAPI,
     order_type: OrderType,
     user_mail: str,
     user_name: str,
 ):
+    # Mock Freshdesk ticket creation to raise TicketCreationError
+    mock_create_ticket.side_effect = TicketCreationError("ERROR")
+
```

Comment:
I'm not a huge fan of using the patch decorators for mocking. It becomes quite confusing when there are multiple of them. Something like this might be clearer. You could alsp extract the instantiation of ticket_response into a smaller function if you prefer.
```suggestion
def test_submit_ticketexception(
    all_orders_to_submit,
    orders_api: OrdersAPI,
    order_type: OrderType,
    user_mail: str,
    user_name: str,
):

# GIVEN the ticketing client providing a valid response
ticket_response=TicketResponse(id=ticket_id,
        email=user_mail,
        priority=1,
        source=2,
        status=2,
        subject="Support needed..",
        created_at=dt.datetime.now(),
        attachments=[],
        tags=[],)

with patch.object(ordes_api.ticket_handler.freshdesk_client, 'create_ticket', return_value=ticket_response) as mock_create_ticket:
    # WHEN submitting an order
    orders_api.submit(
            project=order_type,
            order_in=order_data,
            user_name=user_name,
            user_mail=user_mail,
        )
```